client/my-sites/upgrade-nudge/index.jsx

```diff
@@ -26,7 +26,14 @@ export default React.createClass( {
 		icon: React.PropTypes.string,
 		event: React.PropTypes.string,
 		href: React.PropTypes.string,
-		jetpack: React.PropTypes.bool
+		jetpack: React.PropTypes.bool,
+		feature: React.PropTypes.oneOf( [ 
+			false,
```

Comment:
Can we merge this now and move it later when we have more of these plans-related constants and move them to one place?